### DISCUSSION TITLE
Issue creating Statamic Blog

### DISCUSSION THREAD

**Discussion by CodePranjal** [statamic/cms#9117] [Q&A]
Hello, I don't know if anyone can help or gone through the same problem as I am. I have created a blog post watching laracast statamic tutorial. Created Blog/News inside Pages. And created Blog/News inside collection.
set the index and show template. Called Index template.
but its not working when I go to the "localhost/blog". Nothing shows up!
But I can see the result using slug into browser directly. Means, somehow, Blog could not call the index template.

problem is, i tried so many ways, and search online. But could not get any help!

If anyone can help, really appreciate it. Thanks.

**Comment by duncanmcclean** [comment]
> but its not working when I go to the "localhost/blog". Nothing shows up!

Are you getting a 404 or some other kind of error? Can you take a screenshot of what you're seeing?

**Reply by CodePranjal** [reply]
Hello, Sorry for late reply. I have already solved the issue. Thanks for helping :)

**Reply by jackmcdade** [reply]
Great, glad to hear it!

**Comment by CodePranjal** [comment]
Hi @jackmcdade 
First, I am simply in love with statamic. Best thing I learnt so far :) 
I wanted to ask, if you have any tutorial on how to show more blog post as you have the tutorial for next and previous page. 
Thanks

**Reply by jasonvarga** [reply]
Not a tutorial, but the tags you're looking for are:
- https://statamic.dev/tags/collection-next
- https://statamic.dev/tags/collection-previous

**Reply by CodePranjal** [reply]
Thanks :)

**Reply by CodePranjal** [reply]
hi again @jasonvarga 
I got three questions to ask. 
#1. if there's a way to have categories instead of tags in statamic. or taxonomy is everything? 
#2. is there any way that last blog entry will always go to Feature section?
#3. inside collection => 'pages', can set blueprint for each page?

Thanks in advance.

**Comment by CodePranjal** [comment]
Hello @jackmcdade 
Firstly, wanted to say thanks for statamic. Its so powerful and amazing, the best thing I am learning and using up until now :) 

however, as I am new, got into some stupid issues, like, search queries, perticular nav item to perticular button, etc etc. it would be gr8, if can have more tutorials from you :) 

also, wanted to ask, I have installed font awesome plugin / addon, added the tokens, but everytime i try to access blueprints its loading, not showing anything...

is that a issue with the plugin?

I checked, it authenticate the token bt somehow returns null

        return Cache::rememberForever("font_awesome::kit::{$this->kitToken}", function () {
            $response = dd(Http::withToken($this->authToken())
                ->post('https://api.fontawesome.com', [
                    'query' => $this->kitQuery(),
                ])->json())['data']['me']['kit'];

**Reply by duncanmcclean** [reply]
> however, as I am new, got into some stupid issues, like, search queries, perticular nav item to perticular button, etc etc. it would be gr8, if can have more tutorials from you

Feel free to open new GitHub discussions or ask on the Statamic Discord about any specific issues you're facing.

I'm sure people in the community will be happy to help!

> also, wanted to ask, I have installed font awesome plugin / addon, added the tokens, but everytime i try to access blueprints its loading, not showing anything...
> 
> is that a issue with the plugin?

Sounds like it. You'd be best opening an issue on the addon's GitHub repository.

**Comment by CodePranjal** [comment]


**Reply by edalzell** [reply]
Please open a new discussion, this is an old and unrelated discussion.
